### DISCUSSION TITLE
Cannot resize window - full or min only

### DISCUSSION THREAD

**Discussion by sparker888** [standardnotes/forum#2570] [General]
Just today I can no longer resize the window for SN. I'm actually trying to grab the top bar to pull it to another monitor, but this requires it to resize to do so. I click the arrow to resize and nothing happens. Anyone have similar behavior and/or a fix?

**Comment by JaspalSuri** [comment]
Hi @sparker888,

I'm sorry to hear that. Can you provide us with a few details about your setup? I'm afraid that I wasn't able to reproduce this issue on my end. 

What OS are you running the desktop app on?

What version of the Standard Notes desktop app are you running? The latest version is v3.5.11. You can locate the version of the desktop app that's currently running on your device by clicking on the **Account** menu button and scrolling all the way down.

Are you using any windows tiling manager applications that might interfere with the ability to resize windows?

**Comment by filupmarley** [comment]
Did you try hitting F11?

**Comment by cedewey** [comment]
I'm having the same issue. Running the latest version 3.23.293

I am running Pop! OS 

I'm not running anything other than the default OS behavior in terms window tiling applications. This has been an issue since I installed the app. 

Hitting F11 does not resolve the issue.

**Comment by effieeee** [comment]
Hi @cedewey, may I ask if you're able to take a screen recording of this behavior so far? Also, may I know if resetting the app as instructed in [this link](https://docs.standardnotes.com/troubleshooting/reset-apps/#desktop) might help?

**Comment by cedewey** [comment]
Hi @effieeee,

I deleted all the fields in the Data Directory. Then I deleted all the Backups. Then I went to my app store to uninstall Standard Notes. When I got to the project page within the store, it says there was an update available. So, I went ahead and ran the update. That fixed the issue. I can now resize the window as expected. Thanks!
